### PR TITLE
QRIS Icon

### PR DESCRIPTION
**Pull request by candrasaputra**
** Checklist **
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x] I am confident that all icons are clear and easy to read/understand
- [ ] I have provided a link to the brand icon’s brand guidelines whenever possible.

If this pull request is not adding new icons, you can remove this checklist.

### CODE REVIEW

**Comment by candrasaputra** [comment]
Hi @tauthomas01 , @larouxn please help to review. thank you.

File: app/assets/images/payment_icons/qris.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...itle><path d="M35 0H3a3 3 0 00-3 3v18a3 3 0 003 3h32a3 3 0 003-3V3a3 3 0 00-3-3z" opacity=".07" style="isolation:isolate"/><path d="M35 1a2 2 0 012 2v18a2 2 0 01-2 2H3a2 2 0 01-2-2V3a2 2 0 012-2h32" fill="#fff"/><path fill="#fff" d="M0 0h38v24H0z"/><g clip-path="url(#clip0)" fill="#010101"><path d="M34.607 13.678V10.445H29.21V9.367h5.397V7.212h-8.635V12.6h5.398v1.078h-5.398v2.155h8.635v-2.155zM24.893 7.212h-2.159v8.62h2.16v-8.62zM13.02 7.212v2.155h6.476v1.078H13.02v5.388h2.158v-3.2l3.24 3.2h3.237L18.277 12.6h3.378V7.212h-8.634zM6.543 12.6h2.154v-2.15H6.543v2.15zm.541-1.616h1.079v1.077H7.084v-1.077z"/><path d="M5.465 7.212h-1.89a.273.273 0 00-.249.167.269.269 0 00-.02.103v8.08a.269.269 0 00.27.27h5.127v-2.15H5.465v-6.47zM11.67 7.212H6.544v2.155h3.24V12.6h2.153V7.482a.273.273 0 00-.076-.191.277.277 0 00-.19-.08zM11.942 13.678h-2.16v4.85h2.16v-4.85z"/><path d="M6.318 6H2.27a.271.271 0 00-.27.27v4.04h.54V6.804a.27.27 0 01.27-.264h3.513L6.318 6zM35.46 13.004v3.507a.27.27 0 01-.27.27h-3.512v.534h4.048a.273.273 0 00.253-.166.27.27 0 00.021-.104v-4.041h-.54z"/></g><defs><clipPath id="pi-qris-clip0"><path fill="#fff" transform="translate(2 6)" d="M0 0h34v12.526H0z"/></clipPath></defs></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/qris.svg:1]: It needs the gray border

[video demonstration](https://screenshot.click/03-59-3lc73-sdlgw.mp4)

**Review by tauthomas01 — CHANGES_REQUESTED** [review]
Hi ,

Please follow the [contributions](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) guideline:
- The border of the icon is missing

You can follow this [example icon](https://github.com/activemerchant/payment_icons/blob/master/app/assets/images/payment_icons/visa.svg) as a reference.

**Comment by candrasaputra** [comment]
@tauthomas01 updated

Screen Shot 2021-06-04 at 11 11 47

https://raw.githubusercontent.com/activemerchant/payment_icons/4861c4f3ff5865a38a7332d1cf96bee535853b7f/app/assets/images/payment_icons/qris.svg

**Review by larouxn — APPROVED** [review]
LGTM 🎩 
qris

**Comment by candrasaputra** [comment]
thanks @larouxn 

any ETA for merge?

**Comment by larouxn** [comment]
> thanks @larouxn
> 
> any ETA for merge?

Waiting to hear back from @tauthomas01 as he initially requested changes.

File: app/assets/images/payment_icons/qris.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...<title id="pi-qris">QRIS</title><path fill="#000" opacity=".07" d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3z"/><path fill="#fff" d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32"/><g clip-path="url(#clip0)" fill="#010101"><path d="M34.607 13.678V10.445H29.21V9.367h5.397V7.212h-8.635V12.6h5.398v1.078h-5.398v2.155h8.635v-2.155zM24.893 7.212h-2.159v8.62h2.16v-8.62zM13.02 7.212v2.155h6.476v1.078H13.02v5.388h2.158v-3.2l3.24 3.2h3.237L18.277 12.6h3.378V7.212h-8.634zM6.543 12.6h2.154v-2.15H6.543v2.15zm.541-1.616h1.079v1.077H7.084v-1.077z"/><path d="M5.465 7.212h-1.89a.273.273 0 00-.249.167.269.269 0 00-.02.103v8.08a.269.269 0 00.27.27h5.127v-2.15H5.465v-6.47zM11.67 7.212H6.544v2.155h3.24V12.6h2.153V7.482a.273.273 0 00-.076-.191.277.277 0 00-.19-.08zM11.942 13.678h-2.16v4.85h2.16v-4.85z"/><path d="M6.318 6H2.27a.271.271 0 00-.27.27v4.04h.54V6.804a.27.27 0 01.27-.264h3.513L6.318 6zM35.46 13.004v3.507a.27.27 0 01-.27.27h-3.512v.534h4.048a.273.273 0 00.253-.166.27.27 0 00.021-.104v-4.041h-.54z"/></g><defs><clipPath id="pi-qris-clip0"><path fill="#fff" transform="translate(2 6)" d="M0 0h34v12.526H0z"/></clipPath></defs></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/qris.svg:1]: Correct me if I'm wrong but it doesn't seems like we need the `clipPath`
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 38 24" width="38" height="24" aria-labelledby="pi-qris"><title id="pi-qris">QRIS</title><path fill="#000" opacity=".07" d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3z"/><path fill="#fff" d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32"/><g clip-path="url(#clip0)" fill="#010101"><path d="M34.607 13.678V10.445H29.21V9.367h5.397V7.212h-8.635V12.6h5.398v1.078h-5.398v2.155h8.635v-2.155zM24.893 7.212h-2.159v8.62h2.16v-8.62zM13.02 7.212v2.155h6.476v1.078H13.02v5.388h2.158v-3.2l3.24 3.2h3.237L18.277 12.6h3.378V7.212h-8.634zM6.543 12.6h2.154v-2.15H6.543v2.15zm.541-1.616h1.079v1.077H7.084v-1.077z"/><path d="M5.465 7.212h-1.89a.273.273 0 00-.249.167.269.269 0 00-.02.103v8.08a.269.269 0 00.27.27h5.127v-2.15H5.465v-6.47zM11.67 7.212H6.544v2.155h3.24V12.6h2.153V7.482a.273.273 0 00-.076-.191.277.277 0 00-.19-.08zM11.942 13.678h-2.16v4.85h2.16v-4.85z"/><path d="M6.318 6H2.27a.271.271 0 00-.27.27v4.04h.54V6.804a.27.27 0 01.27-.264h3.513L6.318 6zM35.46 13.004v3.507a.27.27 0 01-.27.27h-3.512v.534h4.048a.273.273 0 00.253-.166.27.27 0 00.021-.104v-4.041h-.54z"/></g></svg>
```

**Review by tauthomas01 — CHANGES_REQUESTED** [review]
Sorry, I was heads down on other priorities.

File: app/assets/images/payment_icons/qris.svg (hunk 2 above)
Comment (candrasaputra) [reply app/assets/images/payment_icons/qris.svg]: you are correct.
thanks.

Comment (candrasaputra) [reply app/assets/images/payment_icons/qris.svg]: @tauthomas01 updated

**Review by tauthomas01 — APPROVED** [review]

File: app/assets/images/payment_icons/qris.svg (hunk 2 above)
Comment (tauthomas01) [reply app/assets/images/payment_icons/qris.svg]: It works! Thank you for making the changes.

**Comment by candrasaputra** [comment]
Hi @tauthomas01 / @larouxn any ETA for deployment? thank youu..

**Comment by larouxn** [comment]
👋 @candrasaputra, hoping to have them live by end of week or early next week.

**Comment by candrasaputra** [comment]
noted, thanks @larouxn

**Comment by larouxn** [comment]
Update, we were able to release a new version of the gem ([v1.7.0](https://github.com/activemerchant/payment_icons/releases/tag/v1.7.0)) but it likely won't be live everywhere until some time next week.

**Comment by candrasaputra** [comment]
noted, thanks @larouxn

**Comment by larouxn** [comment]
👋 Hello @candrasaputra, the icon(s) should be live now.

**Comment by candrasaputra** [comment]
Hi @larouxn thanks a lot. :)

**Pull request merged**